panel/src/config/fiber.js

```diff
@@ -258,19 +258,26 @@ const Fiber = {
   async visit(
     url,
     {
-      replace = false,
+      data = {},
+      headers = {},
+      only = [],
       preserveScroll = false,
       preserveState = false,
-      only = [],
-      headers = {},
-      data = {},
+      replace = false,
+      silent = false
     } = {}
   ) {
     // save the current scrolling positions
     // for all scroll regions
     this.saveScroll();
 
-    document.dispatchEvent(new Event("fiber:start"));
+    if (!silent) {
```

Comment:
Doesn't this mean that the event is never dispatched if `silent` is `true` as Bastian suggested? Or is the event dispatched from elsewhere in this case?

Reply:
https://github.com/getkirby/kirby/pull/3453